frontend/site-registry/src/app/features/details/SiteDetails.tsx

```diff
@@ -155,36 +158,29 @@ const SiteDetails = () => {
 
   const saveSiteDetailsRequestStatus = useSelector(saveRequestStatus);
 
-  useEffect(()=>{
-
-    if(saveSiteDetailsRequestStatus === RequestStatus.success || saveSiteDetailsRequestStatus === RequestStatus.failed)
-    {
-      if(saveSiteDetailsRequestStatus === RequestStatus.success)
-      {
+  useEffect(() => {
+    if (
```

Comment:
The current implementation checks if saveSiteDetailsRequestStatus is either RequestStatus.success or RequestStatus.failed twice. This redundancy can be avoided by using a switch statement or simplifying the conditions.

We use a switch statement to handle different statuses more clearly. This approach is often preferred for handling multiple conditions based on the value of a variable.

Reply:
This is not a part of this functionality, the changes are highlighted due to formatting.